src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -2335,8 +2336,10 @@ private void GetStartedMenuItem_Click(object sender, RoutedEventArgs e)
         /// </summary>
         private void ShowGetStartedGuidedTour()
         {
+            GuideBackground guideBackground = this.FindName("GuidesBackground") as GuideBackground;
```

Comment:
The whole DynamoView is passed to the GuideManager then also you could find the "GuidesBackground" inside the GuidesManager class (using Guide.FindChild) without the need of modifying the GuidesManager constructor and passing it as parameter.

Reply:
@RobertGlobant20 Indeed, worth a try

Reply:
That's right, will update on next commit

Reply:
I am having a bit of trouble with this, the DynamoView is actually passed as a UIElement so to get the overlay element I have to first cast it to DynamoView, which is a bit sketchy to do here in my opinion. Also I couldn't find a FindChild method available here.
![image](https://user-images.githubusercontent.com/66637541/128373739-379b4204-e94f-4fc4-97be-0121d0588c4f.png)
Another solution I can think of would be to save the guideBackground on the DynamoViewModel beforehand to have it accessible here.

@QilongTang, @RobertGlobant20 what are your thoughts on this?

Reply:
hmm, the `guideBackground` is a UI control right? In that way, I wont save it to the view model, maybe we can save it under DynamoView so you dont have to always search for it runtime?